mimic/resource.py

```diff
@@ -84,6 +91,14 @@ def advance_time(self, request):
             "now": seconds_to_timestamp(self.clock.seconds())
         })
 
+    @app.route("/mimic/v1.1/behaviors", branch=True)
```

Comment:
Also, the reason I didn't make it an extra plugin like the nova control plane was that I figured for identity, we wanted a non-random URL just like identity has a non-random URL (since probably we don't necessarily want to make someone auth first to get the service catalog before issuing identity behavior injection requests) - again, I wasn't sure about this though - maybe that's fine?